man/kernel-install.xml

```diff
@@ -42,6 +42,15 @@
       <arg choice="plain">inspect</arg>
       <arg choice="opt"><replaceable>KERNEL-IMAGE</replaceable></arg>
     </cmdsynopsis>
+    <para>For compatibility with kernel <filename>Makefile</filename> invocations:</para>
+    <cmdsynopsis>
+      <command>installkernel</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="plain">VERSION</arg>
+      <arg choice="plain">VMLINUZ</arg>
+      <arg choice="opt">MAP</arg>
+      <arg choice="opt">INSTALLATION-DIR</arg>
+    </cmdsynopsis>
```

Comment:
Hmm, given this is a compat thing primarily I would prefer if this wasn't put front and center here. Can we maybe add this to a new section closer to the bottom of the man page maybe named "Compatibility Invocation" or so? And then do the whole `<cmdsynopsis>` down there instead of up top here